### ISSUE TITLE
In field_validate_instance(), the $update parameter is unused

### ISSUE THREAD

**Issue by docwilmot** [backdrop/backdrop-issues#3151]
## Describe your issue or idea
In `field_validate_instance($instance, $update = FALSE, $load_field = TRUE)`, the `$update` doesnt get used in the function.

**Comment by BWPanda** [comment]
True. Though I don't think its possible to remove it without causing an API change. Adding to the 2.x milestone then.
